packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -3917,7 +4298,7 @@ describe('scope hoisting', function() {
       assert.equal(output.require, 'function');
     });
 
-    it("doesn't support require.resolve calls for included assets", async function() {
+    it.skip("doesn't support require.resolve calls for included assets", async function() {
```

Comment:
Was there a reason for this?

Reply:
Mainly that `require.resolve` isn't really very useful unless you're using the CJS output format, and in that case, if we just pass it through rather than transforming it it already works.

Reply:
> pass it through

But only if it's an external module? `require.resolve("./some-bundled-asset.js")` still wouldn't work

Reply:
Doesn't the current scope hoisting implementation only support external assets anyway? It throws otherwise https://github.com/parcel-bundler/parcel/blob/fdbcbd6996fe3e91014af0e5c18e8ae6ae282a04/packages/shared/scope-hoisting/src/link.js#L773-L777